tests/server/metrics-ga.js

```diff
@@ -0,0 +1,57 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'intern',
+  'intern!object',
+  'intern/chai!assert',
+  'intern/dojo/node!./helpers/init-logging',
+  'intern/dojo/node!fs',
+  'intern/dojo/node!../../server/lib/configuration',
+  'intern/dojo/node!proxyquire'
+], function (intern, registerSuite, assert, initLogging, fs, config, proxyquire) {
+  var suite = {
+    name: 'ga metrics'
+  };
+  var SIGNUP_FLOW = 'Firefox Accounts Sign-up Flow';
+
+  suite['it works with GA'] = function () {
+    var data = JSON.parse(fs.readFileSync('tests/server/fixtures/ga_body_1.json'));
+
+    function analyticsMock() {
+      return {
+        event: function (data) {
+          assert.equal(data.ec, SIGNUP_FLOW);
+          assert.equal(data.ea, 'registered');
+          assert.equal(data.el, 'regular');
+          assert.equal(data.ev, 1);
+          assert.equal(data.hitType, 'event');
+          assert.equal(data.anonymizeIp, 1);
+          assert.equal(data.dataSource, 'web');
+          assert.isDefined(data.ua);
+          assert.equal(data.campaignName, 'none');
+          assert.equal(data.campaignSource, 'none');
+          assert.equal(data.campaignMedium, 'none');
+          assert.isDefined(data.documentHostName, 'none');
+          assert.equal(data.uid, 'c614d7fb-43e4-485c-bf11-40afbb202656');
```

Comment:
yeah, seems like the difference is: client id === identify "session, user id === logged in user. we don't want to save the logged in user so we can use the uuid for both. See http://cutroni.com/blog/2014/04/10/understanding-cross-device-measurement-and-the-user-id/ and https://github.com/peaksandpies/universal-analytics/blob/master/AcceptableParams.md#client-id